app/models/planning_application.rb

```diff
@@ -103,11 +126,52 @@ class PlanningApplication < ApplicationRecord
                            returned], to: :withdrawn, after: proc { |comment|
                                                                update!(cancellation_comment: comment)
                                                              }
+      after do
+        audit("withdrawn", cancellation_comment)
+      end
     end
 
     after_all_transitions :timestamp_status_change # FIXME: https://github.com/aasm/aasm#timestamps
   end
 
+  def audit_created
+    audit("created", nil, Current.user.name)
+  end
+
+  # we already have an auditable module but the arguments need to change. Worth calling super or not?
```

Comment:
why do the arguments need to change?

Reply:
the method in auditable expects an instance variable of @planning_application to be set, in order to grab the id - but its also a controller concern not a model concern - so separate concerns right?